### PR TITLE
add arisusplit keyboard

### PR DESCRIPTION
**Pull request by nouvogaming**
## Description

make split version of arisu

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/arisusplit/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# arisusplit
+
+A split version of the [Arisu](https://github.com/FateNozomi/arisu-pcb) by FateNozomi, in this handwired version, no split backspace version
```
Comment (zvecr) [inline keyboards/arisusplit/readme.md:3]: If this is a handwired board per this line, it should be relocated to `keyboards/handwired`

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by zvecr** [comment]
PR checklist also covers a fair amount of issues, though getting the board moved if required would be preferable so the suggestions dont go stale.

File: keyboards/arisusplit/arisusplit.c [hunk 2]
```diff
@@ -0,0 +1 @@
+#include "arisusplit.h"
```
Comment (drashna) [inline keyboards/arisusplit/arisusplit.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/arisusplit/arisusplit.h [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#pragma once
```
Comment (drashna) [inline keyboards/arisusplit/arisusplit.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/arisusplit/config.h [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x6E67 //ng
+#define PRODUCT_ID      0x6173 //as
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    NouvoKey
+#define PRODUCT         ArisuSplit
+#define DESCRIPTION     Like arisu
```
Comment (drashna) [inline keyboards/arisusplit/config.h:29]: This setting isn't used anymore and should be removed. 

```suggestion
```

File: keyboards/arisusplit/rules.mk [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI controls
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+UNICODE_ENABLE = no         # Unicode
+
```
Comment (drashna) [inline keyboards/arisusplit/rules.mk:33]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth 
AUDIO_ENABLE = no           # Audio output 

```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/arisusplit/rules.mk [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/arisusplit/rules.mk:11]:
```suggestion
```

File: keyboards/arisusplit/rules.mk [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+
+BLUETOOTH_ENABLE = no 		# Enable bluetooth
+AUDIO_ENABLE = no			# Audio Output
+SPLIT_KEYBOARD = yes		# Split keyboard support
```
Comment (fauxpark) [inline keyboards/arisusplit/rules.mk:31]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
SPLIT_KEYBOARD = yes        # Split keyboard support
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**